src/Resources/views/orderCreate.html.twig

```diff
@@ -20,3 +20,50 @@
         'action': path('sylius_admin_order_creation_order_preview', {'customerEmail': app.request.attributes.get('customerEmail')})
     } %}
 {% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+    <script src="{{ asset('bundles/fosjsrouting/js/router.min.js') }}"></script>
+    <script src="{{ path('fos_js_routing_js', { callback: 'fos.Router.setData' }) }}"></script>
+    <script>
+        $(document).ready(function() {
+            $('#sylius_admin_order_creation_new_order_shipments [data-form-collection="add"]').on('click', function(e) {
+                loadAvailableShippingMethods();
+            });
+
+            $('#shipmentsAndPaymentsTab').on('click', function(e) {
+                if ($('#shipmentsAndPayments select').length != 0) {
+                    loadAvailableShippingMethods();
+                }
+            });
+
+            function loadAvailableShippingMethods(form) {
+                form = $('form');
+                form.addClass('loading');
+
+                $.ajax({
+                    type: "POST",
+                    url: Routing.generate(
+                        'sylius_admin_order_creation_ajax_provide_available_shipping_methods',
+                        { 'customerEmail': '{{ app.request.attributes.get("customerEmail") }}', 'shipmentNumber': '0' },
```

Comment:
ShipmentNumber is always equal to 0 - is this behaviour correct?

Reply:
I wanted to make it as simple as it could be (JS is yet sick xD), we could add js support for multiple shipments as an improvement, wdyt?